components/host-dashboard/reports-section/HostFeesSection.js

```diff
@@ -0,0 +1,217 @@
+        } else {
+          return formatCurrency(value, hostCurrency);
+        }
+      },
+    },
+  },
+});
+
+const getHostFeesWithoutShare = (hostFeeNodes, hostFeeShareNodes) => {
+  const totalHostFeeSharePerMonthInCents = hostFeeShareNodes.reduce((result, node) => {
+    const monthKey = new Date(node.date).getMonth();
+    result[monthKey] = (result[monthKey] || 0) + node.amount.valueInCents;
+    return result;
+  }, {});
+
+  return hostFeeNodes.map(node => {
+    const monthKey = new Date(node.date).getMonth();
+    if (totalHostFeeSharePerMonthInCents[monthKey]) {
+      const valueInCents = node.amount.valueInCents - totalHostFeeSharePerMonthInCents[monthKey];
+      return { ...node, amount: { ...node.amount, valueInCents, value: valueInCents / 100 } };
+    } else {
+      return node;
+    }
+  });
+};
+
+const getSeriesFromData = (intl, timeSeries) => {
+  const dataToSeries = data => {
+    const series = new Array(12).fill(0); // = 12 months
+    data?.forEach(({ date, amount }) => (series[new Date(date).getMonth()] = amount.value));
+    return series;
+  };
+
+  const hostFeeNodes = get(timeSeries, 'hostFees.nodes', []);
+  const hostFeeShareNodes = get(timeSeries, 'hostFeeShare.nodes', []);
+  return [
+    { name: 'Host profit', data: dataToSeries(getHostFeesWithoutShare(hostFeeNodes, hostFeeShareNodes)) },
+    ...Object.entries(groupBy(hostFeeShareNodes, 'settlementStatus')).map(([status, nodes]) => ({
+      name: `Host fee share (${i18nTransactionSettlementStatus(intl, status)})`,
+      data: dataToSeries(nodes),
+    })),
+  ];
+};
+
+const getActiveYearsOptions = host => {
+  const currentYear = new Date().getFullYear();
+  const firstYear = host ? parseInt(host.createdAt.split('-')[0]) : currentYear;
+  const activeYears = [...Array(currentYear - firstYear + 1).keys()].map(year => year + firstYear);
+  return activeYears.map(year => ({ value: year, label: year }));
+};
+
+const getQueryVariables = (hostSlug, year) => {
+  return {
+    hostSlug,
+    startDate: `${year}-01-01T00:00:00Z`,
+    endDate: `${year}-12-31T23:59:59Z`,
+  };
+};
+
+const HostFeesSection = ({ hostSlug }) => {
+  const intl = useIntl();
+  const [selectedYear, setSelectedYear] = React.useState(() => new Date().getFullYear());
+  const variables = getQueryVariables(hostSlug, selectedYear);
+  const { loading, data, previousData } = useQuery(mainReportsQuery, { variables, context: API_V2_CONTEXT });
+  const host = loading && !data ? previousData?.host : data?.host;
+  const timeSeries = host?.hostMetricsTimeSeries;
+  const series = React.useMemo(() => getSeriesFromData(intl, timeSeries), [timeSeries]);
+  const yearsOptions = React.useMemo(() => getActiveYearsOptions(host), [host]);
+  const chartOptions = React.useMemo(() => getChartOptions(intl, host?.currency), [host?.currency]);
+
+  if (loading && !host) {
+    return <Loading />;
+  }
+
+  return (
+    <Box py={3} css={{ background: '#F6F5FF' }}>
```

Comment:
```suggestion
    <Box py={3} css={{ background: defaultColors.purple["50"] }}>
```

Reply:
This is supposed to match `colors.primary`, not `colors.purple`. We'll work on theming before releasing, some parts of it we still need to figure out.